common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -142,6 +142,19 @@ def suffixed_pr_message_header
       end
 
       def message_trailers
+        return unless signoff_trailers || custom_trailer
+
+        [signoff_trailers, custom_trailer].compact.join("\n")
+      end
+
+      def custom_trailer
+        trailer = commit_message_options[:trailer]
+        return unless trailer.is_a?(String)
```

Comment:
When would it not be a string?

Reply:
It's mostly a guard clause in case someone would pass say and array of trailers by mistake since the `commit_message_options` object is a plain has that doesn't exactly document which parameters it supports.

Similar like current implementation of sign off details forces it to be a Hash

Reply:
Should we add a test for that? Not worried it won't work, mostly to prevent regressions in the future

Reply:
Makes sense. Or maybe expand a bit and allow both String and Array and raise for anything else? I believe `git commit` command also allows passing `--trailer "token: value"` option multiple times which would add multiple trailers.

Reply:
Yep that makes sense to me. Do you think we should still raise if an invalid option is passed, for example someone might pass a hash thinking it'd be `{ token: "value" }`?

Reply:
I think it makes sense and it is essentially just one additional line of code which helps debugging if someone would have some code that creates trailers dynamically and it would be added as a hash by mistake for example.

Reply:
Or we could go another way and just allow the hash, it maps better to passing one or multiple trailers and automatically forces to not pass multiple trailers with same `key`

Reply:
That also works for me 👍